synternet/chain.json

```diff
@@ -0,0 +1,162 @@
+    },
+    "genesis": {
+      "genesis_url": "https://github.com/Synternet/synternet-chain-releases/blob/main/networks/mainnet/genesis.json"
+    },
+    "versions": [
+      {
+        "name": "v11",
+        "tag": "v0.11.0",
+        "recommended_version": "v0.11.0",
+        "compatible_versions": [
+          "v0.11.0"
+        ],
+        "cosmos_sdk_version": "v0.47.11",
+        "ibc_go_version": "v7.1.0",
+        "consensus": {
+          "type": "cometbft",
+          "version": "v0.37.5"
+        },
+        "height": 1,
+        "binaries": {
+          "linux/amd64": "https://github.com/Synternet/synternet-chain-releases/releases/download/v0.11.0/syntd-linux-amd64-v0.11.0-mainnet"
+        },
+        "next_version_name": "v13"
+      },
+      {
+        "name": "v13",
+        "tag": "v0.13.0",
+        "proposal": 1,
+        "height": TBD,
+        "recommended_version": "v0.13.0",
+        "compatible_versions": [
+          "v0.13.0"
+        ],
+        "cosmos_sdk_version": "v0.47.11",
+        "ibc_go_version": "v7.1.0",
+        "consensus": {
+          "type": "cometbft",
+          "version": "v0.37.5"
+        },
+        "binaries": {
+          "linux/amd64": "https://github.com/Synternet/synternet-chain-releases/releases/download/v0.13.0/syntd-linux-amd64-v0.13.0-mainnet"
+        },
+        "next_version_name": ""
+      }
+    ]
+  },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/synternet/images/synt.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/synternet/images/synt.svg"
+  },
+  "description": "Synternet is the backbone of the Data Layer, a protocol serving as the customizable execution layer between all blockchains. It enables builders to See, Interpret and Act on any data from any chain, supercharging their applications.",
+  "peers": {
+    "seeds": [],
+    "persistent_peers": [
+      {
+        "id": "5410aa1ab020355e2ff031912457e2bb1e7b6c1f",
+        "address": "34.89.155.53:26656"
+      },
+      {
+        "id": "1da54d20c7339713f1d6d28dd2117087dd33d0ca",
+        "address": "cosmos-seed.icycro.org:26656",
+        "provider": "IcyCRO 🧊"
+      },
+      {
+        "id": "fe21dd474640247888fc7c4dce82da8da08a8bfd",
+        "address": "peer-cosmos-hub-01.stakeflow.io:26656",
+        "provider": "Stakeflow"
+      },
+      {
+        "id": "01c0d24922dcdf6f8816ec814a5c3436c5d5fbc5",
+        "address": "65.108.195.29:36656",
+        "provider": "Staketab"
+      },
+      {
+        "id": "28d36c3d45f0208528de3c38f2934ae241bd23e7",
+        "address": "peer-cosmoshub.mms.team:26656",
+        "provider": "MMS"
+      },
+      {
+        "id": "87ccc1dcc0b846fc1623ab9a5ab55682e8e2ad2e",
+        "address": "seed-cosmoshub.freshstaking.com:26656",
+        "provider": "FreshSTAKING"
```

Comment:
@VJurenas who communicates with our validators? We need to get this information from them

Reply:
2eed7e175d204680af243e008e21950f81b8d455 @ 34.89.206.173:26656 (Synternet)
cab3e0b0538e900d57a83ad590a3d228004c63a4 @ 35.225.165.111:26656 (Stakin)
6f76ec0b365da744e893b470eac72a87482359d5 @ 176.9.125.13:2220 (Imperator)
ac6ec924a0501b55db82c6a76aab893aed9a8630 @ 65.109.122.249:26656 (Polkachu)
95b14ec701608044c261ebd15d0b3bd84e295acb @ 72.46.84.135:26656 (Restake)

Reply:
Thanks, updated